pl/whatsnew/whatsnew_android_48.lang

```diff
@@ -18,6 +18,6 @@ Migracja listy „Do przeczytania” do zakładek
 
 # "Snazzy" is an informal English word that signifies "stylish", "attractive".
 ;Video controls get a new look! Snazzy
-Video controls get a new look! Snazzy
+Nowy wygląd elementów sterujących podczas odtwarzania filmów
```

Comment:
This is definitely UTF-8 on my computer. Clicking "View the whole file" on GitHub also shows correct characters.